{
    "data": [
        {
            "test_name": "WorkflowAlgorithmsTest_SofTwoThetaTOFTest",
            "latest_failed_build": "2806",
            "agent_name": "Mac",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 2,
            "past_outcome": {
                "build_keys": [
                    "2806",
                    "2805",
                    "2804",
                    "2803",
                    "2802",
                    "2801",
                    "2800",
                    "2799",
                    "2798",
                    "2797",
                    "2796",
                    "2795",
                    "2794",
                    "2793",
                    "2792",
                    "2791",
                    "2790",
                    "2789",
                    "2788",
                    "2787",
                    "2786",
                    "2785",
                    "2784",
                    "2783",
                    "2782",
                    "2781",
                    "2780",
                    "2779",
                    "2778",
                    "2777"
                ],
                "past_outcome_list": [
                    "Failed",
                    "Failed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "WorkflowAlgorithmsTest_SofTwoThetaTOFTest",
                "build": "2806",
                "agent_name": "Mac",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "0.64",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.6.20230602.1033.dev205\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nDllOpen-[Error] Could not open library /Users/builder/Jenkins/workspace/pull_requests-conda-osx/build/bin/libMantidPythonInterfaceCore.dylib: dlopen(/Users/builder/Jenkins/workspace/pull_requests-conda-osx/build/bin/libMantidPythonInterfaceCore.dylib, 10): Symbol not found: _PyBaseObject_Type\n  Referenced from: /Users/builder/Jenkins/workspace/pull_requests-conda-osx/mambaforge/envs/mantid-developer/lib/libboost_python38.dylib\n  Expected in: flat namespace\n in /Users/builder/Jenkins/workspace/pull_requests-conda-osx/mambaforge/envs/mantid-developer/lib/libboost_python38.dylib\nRunning 4 tests.\nIn SofTwoThetaTOFTest::test_twoThetaGrouping:\n/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/WorkflowAlgorithms/test/SofTwoThetaTOFTest.h:66: Error: Expected (angleBinEdge &lt; twoTheta), found (1.0000000008 &gt;= 0.0028875719)\n/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/WorkflowAlgorithms/test/SofTwoThetaTOFTest.h:66: Error: Expected (angleBinEdge &lt; twoTheta), found (1.0017453290 &gt;= 0.0044857097)\n/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/WorkflowAlgorithms/test/SofTwoThetaTOFTest.h:66: Error: Expected (angleBinEdge &lt; twoTheta), found (1.0034906583 &gt;= 0.0063313326)\n/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/WorkflowAlgorithms/test/SofTwoThetaTOFTest.h:66: Error: Expected (angleBinEdge &lt; twoTheta), found (1.0052359875 &gt;= 0.0079858509)\n/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/WorkflowAlgorithms/test/SofTwoThetaTOFTest.h:66: Error: Expected (angleBinEdge &lt; twoTheta), found (1.0069813178 &gt;= 0.0096397010)\n/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/WorkflowAlgorithms/test/SofTwoThetaTOFTest.h:66: Error: Expected (angleBinEdge &lt; twoTheta), found (1.0087266460 &gt;= 0.0113132258)\n.i: 0 j: 0\ni: 1 j: 0\ni: 2 j: 0\ni: 3 j: 0\ni: 4 j: 0\n\nIn SofTwoThetaTOFTest::test_averaging:\n/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/WorkflowAlgorithms/test/SofTwoThetaTOFTest.h:140: Error: Expected (Ys[j] == 0.), found (2.0000000006 != 0.0000000000)\n/Users/builder/Jenkins/workspace/pull_requests-conda-osx/Framework/WorkflowAlgorithms/test/SofTwoThetaTOFTest.h:141: Error: Expected (Es[j] == 0.), found (1.4142135625 != 0.0000000000)\nFailed 2 of 4 tests\nSuccess rate: 50%\n\n          Start 1296: MuonTest_AsymmetryCalcTest\n"
            }
        },
        {
            "test_name": "python.scripts.AbinsCalculateSPowderTest.AbinsCalculateSPowderTest",
            "latest_failed_build": "2799",
            "agent_name": "Mac",
            "last_failed_outcome": "Failed",
            "pipeline_name": "pull_requests-conda-osx",
            "problem_count": 1,
            "past_outcome": {
                "build_keys": [
                    "2806",
                    "2805",
                    "2804",
                    "2803",
                    "2802",
                    "2801",
                    "2800",
                    "2799",
                    "2798",
                    "2797",
                    "2796",
                    "2795",
                    "2794",
                    "2793",
                    "2792",
                    "2791",
                    "2790",
                    "2789",
                    "2788",
                    "2787",
                    "2786",
                    "2785",
                    "2784",
                    "2783",
                    "2782",
                    "2781",
                    "2780",
                    "2779",
                    "2778",
                    "2777"
                ],
                "past_outcome_list": [
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Passed",
                    "Failed",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None",
                    "None"
                ]
            },
            "last_stack_traces": {
                "test_name": "python.scripts.AbinsCalculateSPowderTest.AbinsCalculateSPowderTest",
                "build": "2799",
                "agent_name": "Mac",
                "trial": 3,
                "outcome": "Failed",
                "test_time": "5.48",
                "stacktrace": "FrameworkManager-[Notice] Welcome to Mantid 6.6.20230602.1033.dev34\nFrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid\nPython-[Notice] Data not found in cache. Structure factors need to be calculated.\nPython-[Notice] Warning: No group Powder in hdf file. Data has to be calculated.\nPython-[Notice] Tensor data for analytic powder averaging has been calculated.\n.Python-[Notice] Data not found in cache. Structure factors need to be calculated.\nPython-[Notice] Warning: [Errno 2] Unable to open file (unable to open file: name = &#x27;Si2-sc_CalculateSPowder.hdf5&#x27;, errno = 2, error message = &#x27;No such file or directory&#x27;, flags = 0, o_flags = 0) Data has to be calculated.\nPython-[Notice] Tensor data for analytic powder averaging has been calculated.\nFPython-[Notice] Data not found in cache. Structure factors need to be calculated.\nPython-[Notice] Warning: [Errno 2] Unable to open file (unable to open file: name = &#x27;Si2-sc_CalculateSPowder.hdf5&#x27;, errno = 2, error message = &#x27;No such file or directory&#x27;, flags = 0, o_flags = 0) Data has to be calculated.\nPython-[Notice] Tensor data for analytic powder averaging has been calculated.\n.Python-[Notice] Using default setting &quot;A&quot; for instrument &quot;MARI&quot;\nPython-[Notice] Using default chopper frequency for instrument MARI: 400 Hz\nPython-[Notice] Data not found in cache. Structure factors need to be calculated.\nPython-[Notice] Warning: [Errno 2] Unable to open file (unable to open file: name = &#x27;Si2-sc_CalculateSPowder.hdf5&#x27;, errno = 2, error message = &#x27;No such file or directory&#x27;, flags = 0, o_flags = 0) Data has to be calculated.\nPython-[Notice] Tensor data for analytic powder averaging has been calculated.\n.Python-[Notice] Using default setting &quot;&quot; for instrument &quot;PANTHER&quot;\nPython-[Notice] Using default setting &quot;&quot; for instrument &quot;PANTHER&quot;\nPython-[Notice] Data not found in cache. Structure factors need to be calculated.\nPython-[Notice] Warning: [Errno 2] Unable to open file (unable to open file: name = &#x27;Si2-sc_CalculateSPowder.hdf5&#x27;, errno = 2, error message = &#x27;No such file or directory&#x27;, flags = 0, o_flags = 0) Data has to be calculated.\nPython-[Notice] Tensor data for analytic powder averaging has been calculated.\n..\n======================================================================\nFAIL: test_1d_order10 (AbinsCalculateSPowderTest.SCalculatorFactoryPowderTest)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/scripts/test/Abins/AbinsCalculateSPowderTest.py&quot;, line 117, in test_1d_order10\n    self._good_case(name=self._si2 + &quot;_1d_o10&quot;, castep_name=self._si2, quantum_order_num=2, autoconvolution=True)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/scripts/test/Abins/AbinsCalculateSPowderTest.py&quot;, line 158, in _good_case\n    self._check_data(good_data=good_data[&quot;S&quot;], data=calculated_data.extract())\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/scripts/test/Abins/AbinsCalculateSPowderTest.py&quot;, line 228, in _check_data\n    assert_almost_equal(ref, calculated)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/mambaforge/envs/mantid-developer/lib/python3.8/site-packages/numpy/testing/_private/utils.py&quot;, line 583, in assert_almost_equal\n    return assert_array_almost_equal(actual, desired, decimal, err_msg)\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/mambaforge/envs/mantid-developer/lib/python3.8/site-packages/numpy/testing/_private/utils.py&quot;, line 1046, in assert_array_almost_equal\n    assert_array_compare(compare, x, y, err_msg=err_msg, verbose=verbose,\n  File &quot;/Users/builder/Jenkins/workspace/pull_requests-conda-osx/mambaforge/envs/mantid-developer/lib/python3.8/site-packages/numpy/testing/_private/utils.py&quot;, line 844, in assert_array_compare\n    raise AssertionError(msg)\nAssertionError: \nArrays are not almost equal to 7 decimals\n\nMismatched elements: 656 / 4100 (16%)\nMax absolute difference: 1.23045312e-06\nMax relative difference: 3.62948515\n x: array([5.7885427e-26, 6.7769479e-26, 4.1057139e-26, ..., 0.0000000e+00,\n       0.0000000e+00, 0.0000000e+00])\n y: array([3.2068142e-25, 3.7467481e-25, 3.4610086e-25, ..., 0.0000000e+00,\n       0.0000000e+00, 0.0000000e+00])\n\n----------------------------------------------------------------------\nRan 6 tests in 2.804s\n\nFAILED (failures=1)\n\n          Start    3: TypesTest_SpectrumDefinitionTest\n"
            }
        }
    ]
}